lib/cloudregister/registerutils.py

```diff
@@ -509,6 +510,47 @@ def get_credentials(credentials_file):
     return (username, password)
 
 
+# ----------------------------------------------------------------------------
+def setup_registry(registry_fqdn, username, password):
```

Comment:
What's the point of this method? Why not just call `set_registry_auth_token`, or rename it `s/set_registry_auth_token/setup_registry/` ?

Reply:
in that method I plan to call more methods, so when `setup_registry` finishes, all the bits needed for registry are done and in place